diff --git a/node_modules/victory-native/src/cartesian/CartesianChart.tsx b/node_modules/victory-native/src/cartesian/CartesianChart.tsx
index 9b82b54..c48c972 100644
--- a/node_modules/victory-native/src/cartesian/CartesianChart.tsx
+++ b/node_modules/victory-native/src/cartesian/CartesianChart.tsx
@@ -451,8 +451,8 @@ function CartesianChartContent<
 
           handleTouch(
             v,
-            touch.absoluteX - scrolledX,
-            touch.absoluteY - scrolledY,
+            touch.x - scrolledX - valueFromSidedNumber(padding, "left"),
+            touch.y - scrolledY - valueFromSidedNumber(padding, "top"),
           );
         } else {
           gestureState.value.bootstrap.push([v, touch]);
@@ -478,8 +478,8 @@ function CartesianChartContent<
 
         handleTouch(
           v,
-          touch.absoluteX - scrolledX,
-          touch.absoluteY - scrolledY,
+          touch.x - scrolledX - valueFromSidedNumber(padding, "left"),
+          touch.y - scrolledY - valueFromSidedNumber(padding, "top"),
         );
       }
     })
@@ -511,8 +511,8 @@ function CartesianChartContent<
 
         handleTouch(
           v,
-          touch.absoluteX - scrolledX,
-          touch.absoluteY - scrolledY,
+          touch.x - scrolledX - valueFromSidedNumber(padding, "left"),
+          touch.y - scrolledY - valueFromSidedNumber(padding, "top"),
         );
       }
     })
